pkg/eib/eib.go

```diff
@@ -211,6 +224,53 @@ func buildCombustion(ctx *image.Context, rootDir string) (*combustion.Combustion
 	return combustionHandler, nil
 }
 
+func generateCombustion(ctx *image.Context, rootDir string) (*combustion.Combustion, error) {
```

Comment:
This brings duplication without an obvious benefit. Especially as `combustion.SkipRPMComponent` already accounts for the new flow. Let's see how this evolves after we apply the rest of the changes.